dashboard/app/controllers/level_sources_controller.rb

```diff
@@ -22,15 +23,15 @@ def show
       # embed the play area and code area for viewing solutions when contributing hints
     else
       # sharing
-      level_view_options hide_source: true
+      level_view_options @level_source.level_id, hide_source: true
```

Comment:
I vote we take this opportunity to do a better job of [enforcing our styleguide](https://github.com/bbatsov/ruby-style-guide#no-dsl-parens) re: parens.